admission/Makefile

```diff
@@ -1,10 +1,10 @@
 CONTROLLER_TOOLS_VERSION = 0.14.0
-KUSTOMIZE_VERSION = 4.5.7
+KUSTOMIZE_VERSION = 5.2.1
 CONTOUR_VERSION = 1.28.2
-ARGOCD_VERSION = 2.10.1
+ARGOCD_VERSION = 2.10.2
```

Comment:
We now use `2.10.7`.

Reply:
I did the merge this morning and could not figure out what was appropriate. 
I will update it.